recipes/rnmc-native/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: rnmc-native
+  version: 1.0
+
+source:
+  url: https://github.com/danielbarter/RNMC_native/archive/2734f5d.zip
+  sha256: 9471e68d211a927381c7a1dda548f50aec4ef62ee42b239b1d11eb72798ee40b
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "./build.sh RNMC"
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - gsl
```

Comment:
You shouldn't need `gsl` as both build & host dependency. It should be enough to have it under `host`.

Reply:
yep, seems to work. Thanks @dopplershift